pep/app/pods/components/search/refine/component.ts

```diff
@@ -30,15 +31,18 @@ interface SearchRefineArgs {
 
 export default class SearchRefine extends Component<SearchRefineArgs> {
     @service intl!: IntlService;
+    @service configuration!: ConfigurationService;
 
     @tracked expandedGroups: string[] = [];
 
     get groups() {
+        const cfg = this.configuration.base.search;
         const groups: RefineGroup[] = [];
         const fieldsMap = this.args.metadata?.facetCounts?.facet_fields ?? {};
         const incFields = Object.keys(fieldsMap);
+        const displayedFacets = cfg.facets.defaultFields.map((id) => SEARCH_FACETS.findBy('id', id)!);
```

Comment:
I have a feeling using `!` is probably the wrong way to go here

Reply:
yeah so its due to TS not knowing how ember's convenience `findBy()` returns, and that it is guaranteed to return an object here.

The alternative to the `!`, is the more verbose:
```
.map((id) => SEARCH_FACETS.find((facet) => facet.id === id));
```
But i'm fine with switching to that if you prefer (just losing the ability to use all of ember's nice convenience methods that we pay for in our vendor payload size is 😢)